packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/text/ReactTextView.java

```diff
@@ -67,6 +67,12 @@ public class ReactTextView extends AppCompatTextView implements ReactCompoundVie
   private ReactViewBackgroundManager mReactBackgroundManager;
   private Spannable mSpanned;
 
+  /**
+   * Used to collect some text size affecting attributes to fix some text cut-off issues when users
+   * adjust text size and font weight to the max value in system font settings.
+   */
+  private TextAttributes mTextAttributes;
```

Comment:
It seems a bit confusing, to have most of the attributes like font size, letter spacing, as member variables, then also having TextAttributes with those same members, and sometimes we use one but not the other.

Could we either:
1. Remove the member variables for where we can use TextAttributes instead?
2. Keep using the member variables, and copy any logic we might be wanting to use?

Reply:
> It seems a bit confusing, to have most of the attributes like font size, letter spacing, as member variables, then also having TextAttributes with those same members, and sometimes we use one but not the other.
> 
> Could we either:
> 
> 1. Remove the member variables for where we can use TextAttributes instead?
> 2. Keep using the member variables, and copy any logic we might be wanting to use?

I'll remove the member variables that not needed now.